backend/Restaurant-System/Handlers/AddTableHandler.cs

```diff
@@ -0,0 +1,26 @@
+﻿using MediatR;
+using Restaurant_System.DataAcessLayer;
+using Restaurant_System.DataAcessLayer.Models;
+
+namespace Restaurant_System.Handlers
+{
+    public class AddTableHandler : IRequestHandler<AddTableCommand>
+    {
+        private readonly IContext _context;
+
+        public AddTableHandler(IContext context)
+        {
+            _context = context;
+        }
+        
+        public Task<Unit> Handle(AddTableCommand request, CancellationToken cancellationToken)
+        {
+            var table = new Table(request.IdRestaurant, request.IdRoom, request.NumberOfSeats, request.IsReserved);
```

Comment:
W kontekście zmian na bazie to też będzie trzeba zmienić. 
Jeśli robisz to w ten sposób możliwe będzie dodanie stolika do nieistniejącej restauracji itd.
Takie rzeczy również trzeba brać pod uwagę.